ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClient.java

```diff
@@ -74,11 +73,12 @@ void createTopic(
   Map<String, TopicDescription> describeTopics(Collection<String> topicNames);
 
   /**
-   * [warn] synchronous call to get the response
+   * [warn] Asynchronous call to get the response
    *
-   * @param topicName topicName to describe
+   * @param topicName topicNames to describe
```

Comment:
Why make this plural. It is still a single topic.

Reply:
This calls `AdminClient.describeTopics` which gets a collection of topics.

Reply:
Yes. But this method just takes a single topic and passes a list with a single element and also assumes the output contains a single pair. 

So it is more appropriate for javadoc for this method to be singular rather than plural.